.github/workflows/release.yml

```diff
@@ -38,31 +39,33 @@ jobs:
           echo "PROJECT_VERSION=${GITHUB_REF#refs/tags/v}" >> "$GITHUB_ENV"
           echo "Version: ${{ env.PROJECT_VERSION }}"
 
+      - name: Check git tag against toml values
+        if: ${{ matrix.job.os != 'windows-2019' }} # windows too weird
```

Comment:
These files are identical among jobs, so just checking on one job, `ubuntu-20.04` for example, is enough.

Reply:
That is what I thought at first, too, but tell me: Won't the publish step for the other `os` still run, creating a release with some missing files?
https://github.com/ikanago/omekasy/blob/a36cd66cc002d59effa34fadf311d7957a842d1c/.github/workflows/release.yml#L89-L95=